phalcon/cache/backend/file.zep

```diff
@@ -11,9 +11,15 @@
  | If you did not receive a copy of the license and are unable to         |
  | obtain it through the world-wide-web, please send an email             |
  | to license@phalconphp.com so we can send you a copy immediately.       |
+ | This source file is subject to the New BSD License that is bundled	 |
+ | with this package in the file docs/LICENSE.txt.						|
+ |																		|
+ | If you did not receive a copy of the license and are unable to		 |
+ | obtain it through the world-wide-web, please send an email			 |
+ | to license@phalconphp.com so we can send you a copy immediately.	   |
  +------------------------------------------------------------------------+
- | Authors: Andres Gutierrez <andres@phalconphp.com>                      |
- |          Eduar Carvajal <eduar@phalconphp.com>                         |
+ | Authors: Andres Gutierrez <andres@phalconphp.com>					  |
+ |		  Eduar Carvajal <eduar@phalconphp.com>						 |
  +------------------------------------------------------------------------+
```

Comment:
Could you please return this back ![screen shot 2018-11-28 at 10 16 18 am](https://user-images.githubusercontent.com/1256298/49137995-b063b080-f2f6-11e8-9d75-cf7d75ce9fe2.png)

Reply:
Whoops :| The burden of cherry-picking commits across major versions.